### ISSUE TITLE
[FlexibleHeader] Cannot scroll through tracking scroll view in VoiceOver

### ISSUE THREAD

**Issue by sread94** [material-components/material-components-ios#1712]
When VoiceOver is turned on, the tracking scroll view of the flexible header is sometimes inaccessible. Only the elements of the scroll view that are on screen can be accessed by swiping through the elements. Using accessibility scroll, all elements of the scroll view can be accessed. This error seems to be related to which scroll view elements are initially visible on the screen.

**Comment by jverkoey** [comment]
Can confirm that is a problem in the catalog when the scroll view has no accessible elements.

**Comment by romoore** [comment]
If the scroll view has no accessible elements, then that's not really a bug with Flexible Header - it's a bug with the scrollview content. I think we can close this issue.  @jverkoey Can you please clarify why this is specific to Flexible Header?

**Comment by romoore** [comment]
@jverkoey Gentle ping.

**Comment by jverkoey** [comment]
Agreed that this is a UIScrollView issue and not specifically related to the flexible header.
